lib/ansible/modules/cloud/amazon/emr_cluster_facts.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+
+DOCUMENTATION = '''
+---
+module: emr_cluster_facts
+short_description: Get information about an Amazon EMR cluster
+description:
+    - Module gets information about an Amazon EMR cluster
+version_added: "2.6"
+requirements: [ boto3 ]
+author:
+    - "Aaron Smith (@slapula)"
+options:
+    id:
+        description:
+            - ID of the EMR cluster.
+        required: false
+        default: None
```

Comment:
default: None is the default, so you can remove this line.

Reply:
Including this was a mistake on my part.  `cluster_id` should be required so a default value (`None` or otherwise) is of no use here.